packages/botonic-react/src/webview.jsx

```diff
@@ -43,7 +43,8 @@ class App extends React.Component {
     const provider = this.state.session.user.provider
     const impId = this.state.session.user.imp_id
     if (provider === PROVIDER.WHATSAPP) {
-      location.href = 'https://wa.me/' + impId
+      const phone_number = this.state.session.user.unformatted_phone_number
```

Comment:
Will we always have this field to read the bot number or only when it is a WhatsApp cloud?

Reply:
this is a particular case where it will be populated only on whatsapp, other providers will get this as null